services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str | None = None,
+    last_name: str | None = None,
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> QuerySet[User]:
+    return User.objects.get(id=user_id)

```

Comment:
Consider handling the `DoesNotExist` exception for `User.objects.get(id=user_id)` to avoid runtime errors if the user with the given ID does not exist.